app/services/cap_payment_checker.rb

```diff
@@ -0,0 +1,43 @@
+class CapPaymentChecker
```

Comment:
Should we rename this to `CheckCapPayment` to match the verb names of other services?